src/createRollupConfig.ts

```diff
@@ -30,9 +31,7 @@ export function createRollupConfig(opts: TsdxOptions) {
     opts.minify !== undefined ? opts.minify : opts.env === 'production';
 
   const outputName = [
-    `${paths.appDist}/${safePackageName(opts.name)}`,
-    opts.format,
-    opts.env,
+    `${paths.appDist}/${opts.format}/index`,
```

Comment:
If we were nice we would automagically update folks package.json for them

Reply:
Meaning? Codemod thingy? It's could be the next possible step after this whole change.